test/demo/contextListener.test.ts

```diff
@@ -91,7 +97,14 @@ describe('Context listeners and broadcasting', () => {
                 }
             });
 
-            test.todo('When calling broadcast from the first app [behavior TBD - do apps receive their own broadcasts?]');
+            test('When calling broadcast from the first app', async () => {
+                await fdc3Remote.broadcast(testAppIdentity, validContext);
+
+                const receivedContexts = await Promise.all(listeners.map(listener => listener.getReceivedContexts()));
+                receivedContexts.forEach((rc)=>console.log(rc));
```

Comment:
Is this logging intentional or leftover debugging? And if intentional, would put spaces around '=>' for consistency